vmdb/app/models/orchestration_template.rb

```diff
@@ -90,6 +93,16 @@ def validate_content(manager = nil)
     "No #{ui_lookup(:model => 'ExtManagementSystem').downcase} is capable to validate the template"
   end
 
+  def validate_format
+    "Cannot validate format because template type is unknown"
```

Comment:
This method is called `validate_format`, but it doesn't appear to actually validate anything.  It just creates a string.  Should this just be a string const?  Or, is there a real format to be validated here?